packages/analytics-js-integrations/__tests__/integrations/Clevertap/browser.test.js

```diff
@@ -0,0 +1,43 @@
+import Clevertap from '../../../src/integrations/Clevertap/browser';
```

Comment:
I saw region in other destinations also, can this be done as generic test?

Reply:
i have added it in our notion doc, let's not do common test in this pr as it differs the purpose of this pr. tests are covered for the changes i have made. will add region common test in other pr which i raised